exercises/practice/complex-numbers/complex_numbers_test.go

```diff
@@ -0,0 +1,127 @@
+package complex
+
+import (
+	"math"
+	"testing"
+)
+
+func TestNumber_Real(t *testing.T) {
+	for _, tt := range realTestCases {
+		t.Run(tt.description, func(t *testing.T) {
+			n := Number{tt.in.a, tt.in.b}
+			if got := n.Real(); got != tt.want {
+				t.Errorf("Number({%v, %v}).Real() = %v, want %v", tt.in.a, tt.in.b, got, tt.want)
```

Comment:
```suggestion
				t.Errorf("Number%+v.Real() = %v, want %v", tt.in, got, tt.want)
```

This makes the output look like `Number{a:2 b:3}.Real()`, which in the context of this exercise is a valid Go expression that a student can copy/paste and use in their program, while something like `Number({2, 3}).Real()` wouldn't really work.

This feedback also applies to the other tests.